src/resources/department/DepartmentList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { List, TextField, TopToolbar, Datagrid } from 'react-admin'
+
+const ListActions = () => <TopToolbar></TopToolbar>
+
+export default function DepartmentList(): React.ReactElement {
+  return (
+    <List actions={<ListActions />}>
```

Comment:
Thanks @TahaKhanAbdalli  - for these reference data lists, we don't need `show` behaviour, since the user can see the name if the entry in the `list` view. We just need `edit` behaviour.  

Hmm, switching to an `edit` form just to edit one string still seems cumbersome.  Is it possible to edit the value in a small popup modal?  If the user is editing multiple reference items, they will still have the list in the background, and will still "have their eyes on" the rows they are going to edit.